From 4e7d744abff6f2076f2c15d0c69cf591e499a22a Mon Sep 17 00:00:00 2001
From: Liu Ying <Ying.Liu@freescale.com>
Date: Thu, 31 Jan 2013 17:05:13 +0800
Subject: [PATCH 2379/2463] ENGR00242672 MX6 local timer:Remove
 enable_wait_mode global variable

This patch removes the unnecessary global variable declaration for
enable_wait_mode.

Signed-off-by: Liu Ying <Ying.Liu@freescale.com>
---
 arch/arm/mach-mx6/localtimer.c |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/arch/arm/mach-mx6/localtimer.c b/arch/arm/mach-mx6/localtimer.c
index 402368b..2fac9fb 100644
--- a/arch/arm/mach-mx6/localtimer.c
+++ b/arch/arm/mach-mx6/localtimer.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2011-2012 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2011-2013 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -24,8 +24,6 @@
 #include <mach/irqs.h>
 #include <mach/hardware.h>
 
-
-extern bool enable_wait_mode;
 /*
  * Setup the local clock events for a CPU.
  */
-- 
1.7.7.4

